ddtrace/tracer.py

```diff
@@ -893,18 +893,16 @@ def shutdown(self, timeout=None):
         # type: (Optional[float]) -> None
         """Shutdown the tracer.
 
-        This will stop the background writer/worker and flush any finished traces in the buffer. The tracer cannot be
-        used for tracing after this method has been called. A new tracer instance is required to continue tracing.
+        This will call shutdown on all SpanProcessors and stop the AgentWriter.
```

Comment:
```suggestion
```

`SpanProcessor`s are not a public API (yet) so we shouldn't be mentioning them in the public docstring